parser-typechecker/src/Unison/Server/Endpoints/FuzzyFind.hs

```diff
@@ -179,29 +173,12 @@ serveFuzzyFind h codebase mayRoot relativePath limit typeWidth query =
             )
             <$> Backend.termListEntry codebase b0 r n
         Backend.FoundTypeRef r -> do
-          te                              <- Backend.typeListEntry codebase r n
-          DefinitionDisplayResults _ ts _ <- Backend.prettyDefinitionsBySuffixes
-            rel
-            root
-            typeWidth
-            (Suffixify True)
-            codebase
-            [HQ.HashOnly $ Reference.toShortHash r]
-          let
-            t  = Map.lookup (Reference.toText r) ts
-            td = case t of
-              Just t -> t
-              Nothing ->
-                TypeDefinition mempty mempty Nothing
-                  . MissingObject
-                  $ Reference.toShortHash r
-            namedType = Backend.typeEntryToNamedType te
-          pure
-            ( a
-            , FoundTypeResult
-              $ FoundType (bestTypeName td) (typeDefinition td) namedType
-            )
-      )
-      refs
+          te <- Backend.typeListEntry codebase r n
+          let namedType = Backend.typeEntryToNamedType te
+          let typeName = Backend.bestNameForType @v ppe (mayDefault typeWidth) r
+          typeHeader <- Backend.typeDeclHeader codebase ppe r
+          let ft = FoundType typeName typeHeader namedType
+          pure (a, FoundTypeResult ft)
```

Comment:
Previously this code was I think for convenience using `prettyDefinitionsBySuffixes`, but I gave it a more direct implementation. (Since we don't want to be fetching and rendering docs for every search result in the finder - that's too expensive)